### PR TITLE
Adds flux detectors to the full core model

### LINKED ISSUE #6: Add detectors to the full core model
This issue can be closed when the fullcore input adds lines that define flux detectors.

### PR DESCRIPTION
**Pull request by robfairh**
This PR fixes #6.

This PR adds the following files:
* axial-detector.png and axial-detector2.png: specify the location of the detector
* axial1.png: axial flux
* fullcore_det1b1.m: detectors output file

This PR modifies the following files:
* README.md: small updates
* auxiliary.py: adds necessary functions to plot the detector outputs.

### CODE REVIEW

File: auxiliary.py [hunk 1]
```diff
@@ -73,11 +73,134 @@ def fullcore():
...
+    total volume where the detector is applied [cm3]
+    """
+
+    det = data.detectors[name]
+    z = [line[0] for line in det.grids['Z']]
+    val = det.tallies
+    vdetector = V/len(z)
+    val = val/vdetector
+
+    # relerr = det.errors
```
Comment (gwenchee) [inline auxiliary.py]: delete commented out code

**Review by gwenchee — CHANGES_REQUESTED** [review]
Besides the commented code, everything else looks good to me. I'll merge once you remove them

File: auxiliary.py (hunk 1 above)
Comment (robfairh) [reply auxiliary.py]: Done!

**Review by gwenchee — APPROVED** [review]
Nice

**Pull request merged**